src/theme/components/number-input.ts

```diff
@@ -0,0 +1,87 @@
+import { numberInputAnatomy as parts } from "@chakra-ui/anatomy";
+import {
+  createMultiStyleConfigHelpers,
+  defineStyle,
+} from "@chakra-ui/styled-system";
+
+const { definePartsStyle, defineMultiStyleConfig } =
+  createMultiStyleConfigHelpers(parts.keys);
+
+const baseStyle = definePartsStyle({
+  addon: {
```

Comment:
Instead of copy pasting all of this styling from the `input` theme, can you import that object and adjust/overwrite properties where necessary. [Chakra's base theme does something similar](https://github.com/chakra-ui/chakra-ui/blob/%40chakra-ui/react%402.4.1/packages/components/theme/src/components/number-input.ts#L8).

Reply:
Updated.